sql/src/test/java/org/apache/druid/sql/calcite/CalciteMultiValueStringQueryTest.java

```diff
@@ -316,6 +316,7 @@ public void testMultiValueStringOverlapFilterNonLiteral()
         ),
         ImmutableList.of(
             new Object[]{"[\"a\",\"b\"]"},
+            new Object[]{"[\"b\",\"c\"]"},
```

Comment:
i suppose this test should have also selected dim2 to make it easier to verify heh, though I'm still kind of confused, what does the array constructor end up doing to an empty mvd row? I started to try to find out but then ran into a wierd problem with the grammar, you can't actually write `array(array(...))` expression because it gets confused with some other junk in there about array literal parsing.. will look into both of these things later and just assume this is correct.